packages/durable-objects/src/types.ts

```diff
@@ -28,3 +28,53 @@ export interface ExchangeRate {
   decimals: number;
   metadata: Record<string, unknown>;
 }
+
+export enum EventSignature {
```

Comment:
You should be able to get these from the NotionalABI object from the @notional-finance/contracts library